### ISSUE TITLE
does not handle file names containing '-'

### ISSUE THREAD

**Issue by kemifl** [reorx/json_include#3]
Include files are ignored if the file name contains hyphens. I think this can be easily fixed by including a hyphen in the INCLUDE_VALUE_PATTERN regex.

**Comment by reorx** [comment]
Yes, I've fixed this, thanks for your suggestion, really glad if this small library makes any help to you :)
